### DISCUSSION TITLE
ERROR FROM CVS

### DISCUSSION THREAD

**Discussion by user6330** [OP-TED/eForms-SDK#684] [Q&A] [answered]
We constantly find this problem from CVS in our platform

**BR-BT-00137-0204**

even creating on https://enotices2.preview.ted.europa.eu/ site, we still find some problem.
We can't generate the xml so can you please give us some tip to go further?

**Comment by bertrand-lorentz** [comment]
The rule BR-BT-00137-0204 only applies if the notice has a group of lots, so if the XML contains something like :
```xml
<cac:ProcurementProjectLot>
    <cbc:ID schemeName="LotsGroup">GLO-...
```

If you get this error for notices in which you do not intend to have a group of lot, you need to make sure that the XML does not include an "empty" group of lots.

If the notice does have a group of lots, this rule checks that the framework maximum value (BT-157-LotsGroup) is less than the sum of the values indicated in the relevant lots.

If you need a more specific answer, please post the problematic XML notice.

This does not seem to be an issue with the SDK itself, so I'm converting this to a discussion.

**Comment by user6330** [comment]
Here is the XML notice. 

I have 3 `<cac:ProcurementProjectLot>`, 2 for **Lot** and 2 for **LotsGroup**

[COMPETITION_pin-cfc-standard_b948a1b7-b6c1-417c-8713-5cdf3ae97a7d-01.txt](https://github.com/OP-TED/eForms-SDK/files/12762190/COMPETITION_pin-cfc-standard_b948a1b7-b6c1-417c-8713-5cdf3ae97a7d-01.txt)

**Reply by YvesJo** [reply, marked as answer]
Rule BR-BT-00137-0204 was known to have an issue with Group of lots and Framework agreements, this has been fixed with SDK 1.9.
Additionally consider that if tenderers are only allowed to submit tenders for a single lot (<cbc:MaximumLotsSubmittedNumeric>1</cbc:MaximumLotsSubmittedNumeric>) and may only win a single (<cbc:MaximumLotsAwardedNumeric>1</cbc:MaximumLotsAwardedNumeric>), then they will never be able neither to submit for a Group of lots are composed of 2 lots at least.
This is a typical question for the helpdesk ted@publications.europa.eu
